USAGE.md

```diff
@@ -24,6 +24,7 @@ sg = SendGridAPIClient(os.environ.get('SENDGRID_API_KEY'))
 * [CLIENTS](#clients)
 * [CONTACTDB](#contactdb)
 * [DEVICES](#devices)
+* [EMAIL ACTIVITY](#email-activity)
```

Comment:
I think we should move this to the [use cases](https://github.com/sendgrid/sendgrid-python/tree/main/use_cases) directory. This file is auto-generated from our [OpenAPI specification](https://github.com/sendgrid/sendgrid-oai).